qCC/ccCommandLineCommands.cpp

```diff
@@ -585,6 +586,134 @@ bool CommandLoad::process(ccCommandLineInterface& cmd)
 	return true;
 }
 
+CommandLoadCommandFile::CommandLoadCommandFile()
+	: ccCommandLineInterface::Command(QObject::tr("CommandFile"), COMMAND_COMMAND_FILE)
+{}
+
+bool CommandLoadCommandFile::process(ccCommandLineInterface& cmd)
+{
+	cmd.print(QObject::tr("[LOADING COMMANDS FROM FILE]"));
+	if (cmd.arguments().empty())
+	{
+		return cmd.error(QObject::tr("Missing parameter: filename after \"-%1\"").arg(COMMAND_COMMAND_FILE));
+	}
+	QString commandFilePath = cmd.arguments().takeFirst();
+
+	//check if file exists
+	if (!QFileInfo::exists(commandFilePath))
+	{
+		return cmd.error(QObject::tr("Command file not exists \"-%1\"").arg(commandFilePath));
+	}
+
+	QFile commandFile(commandFilePath);
+	if (commandFile.open(QIODevice::ReadOnly))
+	{
+		int insertingIndex = 0;
+		QTextStream in(&commandFile);
+		while (!in.atEnd())
+		{
+			QString line = in.readLine();
+			QStringList argumentsInLine = line.split(" ");
```

Comment:
Maybe you could call https://doc.qt.io/qt-6/qstring.html#simplified before doing anything? This might help the process later?

Reply:
Well it will remove whitespaces inside filepath, so i think that will break functionality. Of course it is not normal to have any spaces in path, but i,ve seen it many times, even multiple of them.

Reply:
Indeed 😅